runtime/logging_test.go

```diff
@@ -109,9 +133,12 @@ func TestRequestLogging(t *testing.T) {
 		{
 			"/metrics", "*/*", "HELP go_gc_duration_seconds A summary of the pause duration of garbage collection cycles.", // rest omitted
 		},
-		{ // accept-encoding does not matter for "our" handlers -- they don't compress
+		{ // the data handler can compress the response
 			"/v1/data", "gzip", "{\"result\":{}}",
```

Comment:
So this response is not compressed because it's too small? I'd appreciate a comment here -- looking at the test case in a year or two, I suppose I'll have little idea what should or should not be happening here 🙃

Reply:
Thanks for catching this @srenatus
I've improved the tests, we're capturing now both the compressed request and response and test it.
Should be clearer which test case is for compressed and plain data